### DISCUSSION TITLE
Questions to resolve doubts

### DISCUSSION THREAD

**Discussion by saumil-kapadia-stellantispublic** [callstack/react-native-builder-bob#504] [Q&A]
1. Is this library only project? 
2. Can this be used with native iOS, native Android and angular web app?
3. Is there any sample iOS, Android app which makes use of this library?

**Comment by satya164** [comment]
1. Not sure what you're asking. This project is intended to make react native library development easier, if that's what you mean.
2. This project is intended for React Native. Technically you can use them in native iOS, native Android, and angular web app but that's not really the primary use.
3. https://github.com/callstack/react-native-builder-bob/network/dependents
